src/hotspot/cpu/s390/vm_version_s390.hpp

```diff
@@ -568,7 +568,7 @@ class VM_Version: public Abstract_VM_Version {
   static unsigned long z_SIGSEGV();
 
   // No _features_names[] available on this CPU.
-  static void insert_features_names(char* buf, size_t buflen, uint64_t features = _features) {}
+  static void insert_features_names(char* buf, size_t buflen, unsigned long features[] = _features) {}
```

Comment:
I do not see a reason for this change. How to test the s390 build?

Reply:
From the line 123 from this file --  _features is 
```
  static unsigned long _features[_features_buffer_len];
```

The build is tested in GHA https://github.com/AntonKozlov/crac/actions/runs/5841005695/job/15841780175

Reply:
Here the failure before the change: https://github.com/jankratochvil/crac/actions/runs/5854032342/job/15870802152

Reply:
Thanks, I see now GHA does it test. Yes, I agree your fix is correct.

Reply:
Could you Approve the PR  then?